contrib/functions/go/delete-annotations/main.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/GoogleContainerTools/kpt-functions-sdk/go/fn"
+)
+
+const (
+	annotationKeysLiteral = "annotationKeys"
+	annotationDelimeter   = ","
+)
+
+//nolint
+func main() {
+	if err := fn.AsMain(fn.ResourceListProcessorFunc(deleteAnnotations)); err != nil {
+		os.Exit(1)
+	}
+}
+
+func deleteAnnotations(rl *fn.ResourceList) error {
+
+	var annotationKeys []string
+	annotationKeys, err := getAnnotationKeys(rl.FunctionConfig)
+
+	if err != nil {
+		rl.Results = append(rl.Results, getErrorItem(err.Error()))
+		return err
+	}
+
+	items, err := processResources(rl.Items, annotationKeys)
+	if err != nil {
+		rl.Results = append(rl.Results, getErrorItem(err.Error()))
+		return err
+	}
+
+	for _, resultItem := range items {
+		rl.Results = append(rl.Results, resultItem)
+	}
+
+	return nil
+}
+
+func processResources(objects []*fn.KubeObject, annotationKeys []string) ([]*fn.Result, error) {
+	var resultItems []*fn.Result
+	for _, o := range objects {
+		if o.GetKind() == "" || o.GetName() == "" || o.GetAPIVersion() == "" {
+			continue
+		}
+
+		for _, annotationKey := range annotationKeys {
+			removed, err := o.Remove("metadata", "annotations", annotationKey)
+			if err != nil {
+				return nil, err
+			}
+
+			if removed {
+				itemFilePath := o.GetAnnotations()["internal.config.kubernetes.io/path"]
```

Comment:
There is a helper `o.GetPathAnnotation()`. you can use that.

Reply:
It appears it doesn't have it our golang SDK.
I proposed `GetPathAnnotation` in OSS SDK.
@yuwenma Do we plan to add it in our SDK?